lib/testing_appserver/data/user_data.rb

```diff
@@ -18,7 +20,9 @@ def initialize(user_data = {})
       @pwd =  user_data.fetch(:pwd, secret_data['portal_password_admin'])
       @first_name = user_data.fetch(:first_name, DEFAULT_ADMIN_NAME)
       @last_name = user_data.fetch(:last_name, DEFAULT_ADMIN_LASTNAME)
+      @full_name = "#{@first_name} #{@last_name}"
       @type = user_data.fetch(:type, :admin)
+      @pwd_init = user_data.fetch(:pwd_init, :temporary_pwd)
```

Comment:
I don't like name of this parameter `pwd_init`

Maybe `pwd_generation_type` - better to understand

Reply:
So there are gonna be two parameters which are very similar: `pwd_generation_type` and `generate_pwd`
I decided to rename it to `pwd_set_up_type`